src/components/account/PrivateKey.jsx

```diff
@@ -33,21 +34,24 @@ function PrivateKey({ randkey, toggleModal }) {
       </Row>
 
       <Row>
-        <div className="d-flex gap-1 justify-content-between">
+        <div className="d-flex gap-4 justify-content-between">
           <Button variant="primary" onClick={() => setShow(!show)}>
             <FontAwesomeIcon
               className="me-1"
               icon={show ? faEyeSlash : faEye}
             />
             {show ? 'Hide' : 'Show'}
           </Button>{' '}
+          {/* Copying to clipboard does NOT work on HTTP, only HTTPS */}
+          { ENV.BASE_URL.startsWith("https://") ? 
           <Button variant={copySuccess ? 'success' : 'light'} onClick={copy}>
             <FontAwesomeIcon
               className="me-1"
               icon={copySuccess ? faCheck : faCopy}
             />
             {copySuccess ? 'Copied!' : 'Copy'}
           </Button>
+          : <></> }
```

Comment:
You can return `null` instead of an empty fragment. You can shorten this to 

```jsx
{ENV.BASE_URL.startsWith("https://") &&
(<Button variant={copySuccess ? 'success' : 'light'} onClick={copy}>
            <FontAwesomeIcon
              className="me-1"
              icon={copySuccess ? faCheck : faCopy}
            />
            {copySuccess ? 'Copied!' : 'Copy'}
          </Button>)
}
```